app/build.gradle

```diff
@@ -33,6 +34,11 @@ android {
         versionCode 1
         versionName "0.1.6"
         vectorDrawables.useSupportLibrary true
+        javaCompileOptions {
+            annotationProcessorOptions {
+                arguments = ["room.incremental" : "true"]
```

Comment:
Yes, if you do not do this, there will be a warning: [WARN] Incremental annotation processing requested, but support is disabled because the following processors are not incremental: androidx.room.RoomProcessor (DYNAMIC).
If you use this code, the warning will disappear and everything will work correctly: https://stackoverflow.com/questions/57670510/how-to-get-rid-of-incremental-annotation-processing-requested-warning

Reply:
Right now we don't want to make the switch - see https://issuetracker.google.com/issues/112110217 for more details.